internal/restyutil/restyutil_test.go

```diff
@@ -0,0 +1,138 @@
+package restyutil_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/bitcoin-sv/spv-wallet-go-client/config"
+	goclienterr "github.com/bitcoin-sv/spv-wallet-go-client/errors"
+	"github.com/bitcoin-sv/spv-wallet-go-client/internal/restyutil"
+	"github.com/bitcoin-sv/spv-wallet/models"
+	"github.com/go-resty/resty/v2"
+	"github.com/jarcoal/httpmock"
+	"github.com/stretchr/testify/require"
+)
+
+/*
+ What is the test doing?
```

Comment:
This is pointless to add such a comments on tests.

Reply:
yes i was just explaining here what this test do .. i knew that we should remove it, before merge it